dali/operators/decoder/audio/utils_test.cc

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include <vector>
+#include "dali/operators/decoder/audio/utils.h"
+
+namespace dali {
+
```

Comment:
There are no tests for resampling!
I've actually posted a test code on slack - it generates an artificial waveform in two sampling rates and compares the resampled signal to the ground-truth (the signal generated _de novo_ in the new rate). IMO it's the best test we can have, since it doesn't attempt to replicate the results (including artifacts) of some other implementation, but compares against the perfect result instead.